### ISSUE TITLE
`this.handleLoad` isn't a function

### ISSUE THREAD

**Issue by baptisteArno** [ryanseddon/react-frame-component#234]
When updating from 5.2.4 to 5.2.5

**Comment by ryanseddon** [comment]
Oh no, i don't see the error in any tests or on the demo page? Can you create a test case?

**Comment by kevinfaveri** [comment]
same happening to me. Normal usage, but the child component has multiple iframes inside of it

**Comment by janszybowski** [comment]
@ryanseddon Hi, first of all thank you for this lib. Same happens for me. After transforming the code with babel it looks like this
```js
    var _this = _possibleConstructorReturn(this, (Frame.__proto__ || Object.getPrototypeOf(Frame)).call(this, props, context));

    _this.setRef = function (node) {
      _this.nodeRef.current = node;
      var forwardedRef = _this.props.forwardedRef; // eslint-disable-line react/prop-types

      if (typeof forwardedRef === 'function') {
        forwardedRef(node);
      } else if (forwardedRef) {
        forwardedRef.current = node;
      }
    };

    _this.handleLoad = function () {
      clearInterval(_this.loadCheck); // Bail update as some browsers will trigger on both DOMContentLoaded & onLoad ala firefox

      if (!_this.state.iframeLoaded) {
        _this.setState({
          iframeLoaded: true
        });
      }
    };

    _this.loadCheck = setInterval(function loadCheckCallback() {
      this.handleLoad();
    }, 500);
    _this._isMounted = false;
    _this.nodeRef = _react2.default.createRef();
    _this.state = {
      iframeLoaded: false
    };
    return _this;
  }
```
As you can see `this` has been renamed to `_this` but not when it comes to `this.handleLoad();` call.

So this code
```js
loadCheck = setInterval(function loadCheckCallback() {
    this.handleLoad();
  }, 500);
```
shoud look like this
```js
loadCheck = setInterval(() => {
    this.handleLoad();
  }, 500);
```

**Comment by ryanseddon** [comment]
Ok just leaving notes for myself and future readers:

I can reproduce the error with any simple usage! 

Looking at the stack trace I can see `this.handleLoad` is trying to be called but in this case `this` is window and not the `Frame` instance like it should be.

Digging into the compiled src that ships in npm it seems that maybe babel doesn't correctly reference `this` as the compiled code converts all references to this -> `_this`.

https://npmfs.com/package/react-frame-component/5.2.5/lib/Frame.js#L71

**Comment by ryanseddon** [comment]
I've deprecated 5.2.5 for now on npm, but I think the fix might be to upgrade babel I did try migrating to v7 but still it left the this unchanged in the compiled code?

I'm on vacation until the end of the month so can only look at this sporadically.

If anyone has a fix happy to review an publish when I get time.

**Comment by Roman-Maksimov** [comment]
I'm facing the same with very basic usage on **React@18.0.26**

```javascript
<Frame>
    <div>Viewport</div>
</Frame>
```

<img width="484" alt="image" src="https://user-images.githubusercontent.com/8796399/212002016-beb25f2d-0795-4892-906b-86cc5aea7768.png">

**Comment by dannysindra** [comment]
Hi folks, I created a PR to fix the issue - please have a look whenever you get a chance ^

@ryanseddon

**Comment by ryanseddon** [comment]
Thanks everyone, and especially @dannysindra for doing the PR. Just released [v5.2.6](https://github.com/ryanseddon/react-frame-component/releases/tag/v5.2.6)
